src/languageSupport/languages/flux/monaco.flux.syntax.ts

```diff
@@ -1,4 +1,9 @@
 import register from 'src/languageSupport/monaco.onigasm'
+import * as monaco from 'monaco-editor/esm/vs/editor/editor.api'
+
+if (!window.monaco) {
+  window.monaco = monaco
+}
```

Comment:
There is longer a global monaco instance on window.monaco.
https://github.com/TypeFox/monaco-languageclient/pull/248#issuecomment-774026638